server/protocols/jmap/src/main/java/org/apache/james/jmap/model/Emailer.java

```diff
@@ -31,6 +33,8 @@
 @JsonDeserialize(builder = Emailer.Builder.class)
 public class Emailer {
 
+    public static String INVALID = "invalid";
```

Comment:
from, to, etc are optional. What about returning a null (in json, Optional in java) value so that it's the frontend that display a correct and i18n value ?